ydb/public/tools/lib/cmds/__init__.py

```diff
@@ -374,6 +398,11 @@ def deploy(arguments):
         **optionals
     )
 
+    if os.path.exists(os.path.join(arguments.ydb_working_dir, "kikimr_configs/config.yaml")):
```

Comment:
Is it really a good contract to rely on **specific folder name** + **specific file name** instead of, for example, `custom_config_path` argument? 
also `kikimr_configs` means there's could be more than one config, probably you config should have less exclusive name

Reply:
Это плохой контракт, да

"for example, custom_config_path argument" -- это норм